charts/tekton-apps/templates/wordpress/argocd_applications.yaml

```diff
@@ -188,10 +188,8 @@ spec:
         {{- end }}
 
         livenessProbe:
-          httpGet:
-            path: /wp-admin/install.php
+          tcpSocket:
```

Comment:
Doesn't look like a worth change. TCP port can be open, but application can return 500 error on HTTP request. 

Probably, we should expose `healthcheckUrl` parameter and ask developers to implement it in project - just like we do for `GET /livez` endpoints for other backends.

Reply:
@dmitry-mightydevops we discussed this with Dariya on a call. New version of helmchart uses `tcpSocket` for liveness probe. So if we define probe here as `httpGet` then we get a conflict - helmchart mixes both values and we get invalid probe definition.

Originally it was put here to override default period (10s), which cretes signifnicant CPU consumption even for stale wordpress applications without any activity. So we can override livenss/readness checks period only and leave checks intact - as is in helmchart. Then liveness checks TCP, while readness performs `httpGet`. IMHO, that will be enough.

Reply:
`customLivenessProbe` might be used
https://github.com/bitnami/charts/blob/dd40e6e3dc01ef62a9e631540e660416bc6cadf6/bitnami/wordpress/templates/deployment.yaml#L304C11-L304C48
![image](https://github.com/user-attachments/assets/d448e31b-9c10-464a-a89e-6ba59a8a0454)

Reply:
@darliiin @populov see comment from Misha above

Reply:
add customLivenessProbe

Reply:
@darliiin can we use a different check that install.php? Maybe there is some light *from cpu usage perspective URL?

Reply:
@dmitry-mightydevops 

I think we can use  `/wp-login.php`
![image](https://github.com/user-attachments/assets/609ca027-24fb-4e33-8d8c-6d584df6c228)

Reply:
@darliiin ok let's use it. Thx for the suggestion!